docs/features/models/ollama.md

```diff
@@ -10,6 +10,7 @@ title: Ollama
 
     - To download Ollama: https://ollama.com/download
     - To install the ollama python sdk: `pip install outlines[ollama]`
+    - To make sure the Ollama server is running: `ollama serve`
```

Comment:
You don't have to do that to use the `Ollama` model, the client launches it in the background by itself.